src/ppom-controller.ts

```diff
@@ -994,9 +1002,11 @@ export class PPOMController extends BaseControllerV2<
       // thus it is added here to prevent validation from failing.
       await this.#initialisePPOM();
       const { chainStatus } = this.state;
-      const versionInfo =
-        chainStatus[chainId]?.versionInfo ??
-        this.state.versionInfo.filter(({ chainId: id }) => id === chainId);
+      console.log('chainStatus 1008', chainStatus);
+      const hasVersionInfo = chainStatus[chainId]?.versionInfo?.length;
```

Comment:
I noticed this part wasn't been called in tests because chainStatus[chainId]?.versionInfo is usually [] not undefined/null. So this check makes sure it works either it's undefined, null or []